docs/usage/ruby.md

```diff
@@ -17,7 +17,10 @@ Renovate supports upgrading dependencies in Bundler's Gemfiles and their accompa
 
 ## Warnings
 
-Renovate doesn't update dependencies without a version constraint.
+When using `"rangeStrategy": "update-lockfile"`, all gems listed in the `Gemfile.lock` will be updated.
```

Comment:
i think it will only update gems listed in `Gemfile` ( and transitive deps of cause)

Reply:
Hm, when looking at our project where we use Renovate, it does seem so, that it only looks for updates for the Gems listed in the Gemfile. I adapted the README!